_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp

```diff
@@ -988,6 +988,16 @@ mfxStatus VideoDECODEAV1::SubmitFrame(mfxBitstream* bs, mfxFrameSurface1* surfac
                 return MFX_ERR_NONE;
             }
 
+            if (umcRes == UMC::UMC_ERR_FAILED)
```

Comment:
I would not allow code at line 980 an below execute on the error. I would move your code before line 972 and actually reused error handling at lines 974-977.

Reply:
Done